### PR TITLE
fix: add name to local storage and remove whitespaces from both ends of name

### PR DESCRIPTION
**Pull request by a-vishnoi**

### CODE REVIEW

File: src/views/components/ChangeName.jsx [hunk 1]
```diff
@@ -49,8 +50,14 @@ export const ChangeName = ({ showChangeNameModal, setShowChangeNameModal }) => {
   const hmsActions = useHMSActions();
   const [currentName, setCurrentName] = useState("");
   const changeName = async () => {
+    const name = currentName.trim();
+    if (name.length < 1) {
```
Comment (triptu) [inline src/views/components/ChangeName.jsx:54]: is this needed with the disabled condition fix?

Comment (a-vishnoi) [reply src/views/components/ChangeName.jsx:54]: This is required for the case when user presses enter with spaces in name.

Comment (raviteja83) [reply src/views/components/ChangeName.jsx:54]: can we also add onKeyDown on input for enter key

Comment (triptu) [reply src/views/components/ChangeName.jsx:54]: enter key is handled through form submit

**Review by triptu — APPROVED** [review]

**Pull request merged**